packages/story-editor/src/components/library/panes/media/local/mediaRecording/modal.js

```diff
@@ -201,7 +370,31 @@ function Modal({ isOpen, onClose }) {
         name={audioToggleId}
         checked={enableAudio}
         onChange={() => setEnableAudio((value) => !value)}
+        disabled={!audioInput}
       />
+
+      {videoInputs.length > 0 && (
+        <DropDown
+          ariaLabel={__('Video Input', 'web-stories')}
+          placeholder={__('Select Video Input', 'web-stories')}
+          options={videoInputs}
+          onMenuItemClick={onChangeVideoInput}
+          selectedValue={videoInput}
+          disabled={!enableVideo}
```

Comment:
```suggestion
          disabled={!enableVideo || isRecording}
```
Or we could do this.